action.yml

```diff
@@ -99,23 +100,23 @@ runs:
         fi
         git checkout -b "$BRANCH_NAME"
 
-        echo "${{ steps.check-lts.outputs.latest-patch-version }}" > .go-version
+        echo "${{ steps.check-stable.outputs.latest-patch-version }}" > .go-version
 
         find . -name go.mod -execdir sh -c 'awk "/golang.org\/x/ {print \$1}" go.mod | grep "golang.org/x" | while read -r line; do go get $line@latest; done' \;
 
         find . -name go.mod -execdir go mod tidy \;
 
         git add .
-        git commit -m "Bump go version to ${{ steps.check-lts.outputs.latest-patch-version }}"
+        git commit -m "build(deps): bump go version to ${{ steps.check-stable.outputs.latest-patch-version }}"
```

Comment:
~ Would be nice to use conventional commit format (again, similar to Dependabot)